crates/core/tedge_agent/src/tedge_to_te_converter/converter.rs

```diff
@@ -30,9 +30,6 @@ impl TedgetoTeConverter {
             }
             topic if topic.name.starts_with("tedge/events") => self.convert_event(message),
             topic if topic.name.starts_with("tedge/alarms") => self.convert_alarm(message),
-            topic if topic.name.starts_with("tedge/health") => {
```

Comment:
I understand that this was removed to avoid a certain bug discussed in an earlier comment. But, I'm wondering about the impact of this change on backward compatibility. I was of the impression that these conversions exist to make the transition of third-party component written against the old APIs smoother on our next release where they'll continue publishing to the older topics and we'll internally map them with this `tedge_to_te_converter`. So, removing that conversion support for `tedge/health` will break all thirdy-part components publishing to the old health topics.

But, if `tedge/health` was only an internal API in the earlier releases, meant only to be used by our own components and not by third-party components, then this is okay. I'm just not sure if it was just an internal API.

Reply:
That's true that we could only exclude a white-list of tedge daemons. However I don't think this API has been used by many as it was not documented beyond tedge daemons.